src/Pod.ts

```diff
@@ -282,10 +287,18 @@ export default class Pod {
       const rejectNonceIndex = rejectNonces.indexOf(tx.nonce);
       // If there is, we package that together with the regular transaction.
       if (rejectNonceIndex >= 0) {
-        return new Proposal(this, nonce, tx, rejectTransactions[rejectNonceIndex]);
+        // Check to see if the reject transaction has been executed
```

Comment:
Is it possible to move this logic into the Proposal class, I'd prefer not to change the Proposal constructor signature if possible.